mlrun/runtimes/base.py

```diff
@@ -804,7 +813,7 @@ def _delete_resources(
         pass
 
     def _is_crd_object_in_transient_state(
-        self, db: DBInterface, db_session: Session, crd_object
+        self, db: DBInterface, db_session: Session, crd_object, grace_period: int
```

Comment:
Why is grace_period a parameter for discerning whether some object is in a transient state?
Isn't the grace period effect taking hold "above" that?